src/vip_client/__init__.py

```diff
@@ -1,15 +0,0 @@
-"""
-Python client for the Virtual Imaging Platform (VIP): https://vip.creatis.insa-lyon.fr/.
-
-Python classes and methods to interact with VIP through its RESTful API.
-Main user class: VipSession 
-    from vip_client import VipSession
-For more information: https://github.com/virtual-imaging-platform/VIP-python-client.
-"""
-
-# Informations
-__version__ = "0.1.9"
-__license__ = "CECILL-B"
-
-from vip_client.classes import VipSession
-from vip_client.classes import VipGirder
```

Comment:
Should we not keep the 2 imports ?
I think that without them the imports like `from vip_client import VipSession` will fail.